Indicators/TomDemarkSequential.cs

```diff
@@ -87,6 +89,18 @@ public class TomDemarkSequential : WindowIndicator<IBaseDataBar>
     /// </summary>
     public decimal SupportPrice => _supportPrice;
     
+    /// <summary>
+    /// Gets the current Setup step count in the active Tom Demark Sequential (Buy/Sell) Setup phase 
+    /// (e.g., 1 to 9 in Setup), 0 if not in setup phase.
+    /// </summary>
+    public int SetupPhaseStepCount => _setupStepCount;
+    
+    /// <summary>
+    /// Gets the current Countdown step count in the active TomDemark Sequential (Buy/Sell) Countdown phase
+    /// (e.g., 1 to 13 in Setup), 0 if bar is not in Countdown phase.
+    /// </summary>
+    public int CountdownPhaseStepCount => _countdownStepCount;
```

Comment:
minor might be simpler/cleaner if the property just had a public get and a private set, no need to define the backing filed manually for these -> will be able to delete a few lines